gnd/src/test/java/com/google/android/gnd/persistence/local/LocalDataStoreTest.java

```diff
@@ -231,6 +232,39 @@ public void testDeleteProject() {
     localDataStore.getProjects().test().assertValue(AbstractCollection::isEmpty);
   }
 
+  @Test
+  public void testRemovedLayerFromProject() {
+    Layer layer1 =
+        Layer.newBuilder()
+            .setId("layer 1")
+            .setDefaultStyle(Style.builder().setColor("000").build())
+            .build();
+    Layer layer2 =
+        Layer.newBuilder()
+            .setId("layer 2")
+            .setDefaultStyle(Style.builder().setColor("000").build())
+            .build();
+
+    Builder project =
```

Comment:
Nit: Please qualify `Builder` with `Project` instead - we'll often have more than one class called `Builder` imported at a time; by always qualifying by the outer class usage will be more consistent. Thanks!

Reply:
Done